src/chainparamsbase.cpp

```diff
@@ -52,7 +52,7 @@ std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain
     } else if (chain == CBaseChainParams::REGTEST) {
         return MakeUnique<CBaseChainParams>("regtest", 18443, 18445);
     }
-    throw std::runtime_error(strprintf("%s: Unknown chain %s.", __func__, chain));
+    throw std::runtime_error(strprintf("%s: Unknown chain: %s.", __func__, chain));
```

Comment:
maybe suggest valid values to the user
```suggestion
    throw std::runtime_error(strprintf("%s: Unknown chain: %s. Please provide one of: main, test, signet, regtest", __func__, chain));
```